scripts/zones/Windurst_Walls/npcs/Festive_Moogle.lua

```diff
@@ -5,30 +5,57 @@
 --  !pos -206.927 -0.521 -144.713 239
 -----------------------------------
 require("scripts/globals/npc_util")
-
-local stock = {
-    16119,  -- Nomad Cap
-    16118,  -- Moogle Cap
-    18401,  -- Moogle Rod
-    320,    -- Harpsichord
-    264,    -- Stuffed Chocobo
-    11290,  -- Tidal Talisman
-    11811,  -- Destrier Beret
-    10293,  -- Chocobo Shirt
-}
+-----------------------------------
 
 function onTrade(player,npc,trade)
+    --TODO: trade of pellet for prize
 end
 
 function onTrigger(player,npc)
-    player:startEvent(503, unpack(stock))
+    local festiveItems = {getFestiveItems(player)}
+    if festiveItems[1] then
+        player:startEvent(380, unpack(festiveItems))
+    else
+        player:startEvent(381)
```

Comment:
you've used the same csid at all 3 npc's. They do not use the same ID. the correct default event for the one in windurst is 504.

Reply:
my bad. i copy and pasted it after making the changes, will fix this =)